src/features/fingerprinting-canvas.js

```diff
@@ -29,8 +29,11 @@ export default class FingerprintingCanvas extends ContentFeature {
             clearCache(canvas)
         }
 
+        // proxy methods are bound to the handler, so we need a closure reference to `this`
+        const addDebugFlag = this.addDebugFlag.bind(this)
         const proxy = new DDGProxy(featureName, HTMLCanvasElement.prototype, 'getContext', {
             apply (target, thisArg, args) {
+                addDebugFlag()
```

Comment:
I didn't instrument `DDGProxy` because chances are that it will be hidden under `.wrapMethod`, and therefore won't need the debug flags

Reply:
This makes all those a second class citizen despite being more mature. I think we should fix them tbh.